test/routes/entry-point.test.ts

```diff
@@ -0,0 +1,29 @@
+import jsonld from 'jsonld';
+import entryPoint from '../../src/routes/entry-point';
+import createContext from './context';
+
+describe('entry-point', (): void => {
+  it('should return the entry point', async (): Promise<void> => {
+    const context = createContext('entry-point');
+    const next = jest.fn();
+
+    await entryPoint(context.router)(context, next);
+
+    const { response } = context;
+
+    expect(response.status).toEqual(200);
```

Comment:
I don't think it's wrong to use `toEqual` here, but it's usually used to test deep equality. For primitive values, `toBe` might be slightly better (as on [line 23](https://github.com/libero/article-store/pull/10/files#diff-2923edc37cc590e9eebebcf1fac9b18dR23)).